transform/mattermost-analytics/models/marts/web_app/platform/eg_performance_events.sql

```diff
@@ -0,0 +1,29 @@
+{{
+    config({
+        "cluster_by": ['event_date']
+    })
+}}
+
+WITH performance_rc AS (
+    SELECT 
+        'int_mm_telemetry_rc__performance_events' AS _source_relation,
+        {{dbt_utils.star(ref('int_mm_telemetry_rc__performance_events'))}}
+     FROM
+       {{ ref('int_mm_telemetry_rc__performance_events') }}
+{% if is_incremental() %}
+    WHERE received_at_date >= (SELECT MAX(received_at_date) FROM {{ this }}) 
+    WHERE _source_relation = 'int_mm_telemetry_rc__performance_events'
```

Comment:
The `)` is at the wrong place, adding two `WHERE` in the parent statement.

Reply:
Fixed this and the dbt compile was successful